models/User.models.js

```diff
@@ -1,48 +1,39 @@
 const { DataTypes } = require('sequelize');
-const bcrpyt = require('bcrypt');
-
 const db = require('../config/db');
 
-const Customer = db.define('customer', {
-    id_C: {
+const User = db.define('User', {
+    userID: {
         type: DataTypes.INTEGER({length:10}),
         primaryKey: true,
-        autoIncrement: true
+        autoIncrement: true,
     },
-    name_C: {
+    name_User: {
         type: DataTypes.STRING(30),
         allowNull: false
     },
-    username_C: {
-        type: DataTypes.STRING(30),
-        allowNull: false,
-        unique: {
-            msg: "Ese usuario ya existe"
-        }
-    },
-    address_C: {
+    address_User: {
         type: DataTypes.STRING(50),
         allowNull: false
     },
-    numberPhone_C: {
+    user_Phone: {
         type: DataTypes.STRING(10),
         allowNull: false
     },
-    password_C: {
+    type_User: {
+        type: DataTypes.INTEGER({length:5}),
+        allowNull: false
+    },
+    username: {
+        type: DataTypes.STRING(30),
+        allowNull: false,
+        unique: {
+            msg: "Ese usuario ya existe"
+        }
+    },
+    password: {
         type: DataTypes.STRING(60),
         allowNull: false
     }
 }, {
     timestamps: false,
-    hooks: {
```

Comment:
That is the part that allows to hash passwords, why did you delete it?

Reply:
Es para que tu la pongas por que yono puse eso asi que te toca a ti

Reply:
Alright.